src/containers/BooksForm.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable no-restricted-syntax */
+import React, { Component } from 'react';
+
+class BooksForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      bookGenres: ['Action and Adventure', 'Anthology', 'Chick lit', 'Crime', 'Drama', 'Fantasy'],
```

Comment:
Your list of categories will be in a normal `const` data type and not in the `state`. What should go into the `state` are `title` and `category` 

```
this.state = {
      title: '',
      category: 'Action',
    };
```